capstone/src/main/webapp/js/maps.js

```diff
@@ -0,0 +1,66 @@
+/* eslint no-unused-vars: ["error", {"varsIgnorePattern": "fetchData"}] */
+
+/**
+ * Function to fetch data from servlet as JSON and calls createMap().
+ */
+async function fetchData() {
+  const queryUrl = new URL(window.location.href);
+  const ids = queryUrl.searchParams.get('id');
+  await fetch('/maps?id=' + ids)
+      .then((response) => response.json())
+      .then((mapsJsonData) => {
+        console.log('Fetching data...');
+        console.log(mapsJsonData);
+        createMap(mapsJsonData);
+      });
+}
+
+/**
+ * Function that creates map markers and info windows.
+ * @param {JSON} mapsDataFromFetch JSON response from servlet
+ */
+function createMap(mapsDataFromFetch) {
+  const map = new google.maps.Map(
+      document.getElementById('map'),
+      {center: {lat: 39.8097343, lng: -98.5556199}, zoom: 5});
```

Comment:
Might be worth a comment for future readers about why this is the center and zoom level...